packages/amplify-category-function/src/index.ts

```diff
@@ -11,6 +11,7 @@ import { postEnvRemoveHandler } from './events/postEnvRemoveHandler';
 import { postPushHandler } from './events/postPushHandler';
 import { preExportHandler } from './events/preExportHandler';
 import { prePushHandler } from './events/prePushHandler';
+import { ensureLambdaExecutionRoleOutputs } from './provider-utils/awscloudformation/utils/ensure-lambda-arn-outputs';
```

Comment:
No need to import `ensureLambdaExecutionRoleOutputs` if it's just being exported.

```suggestion
```